docs/Modpacks/Ore-Generation/Bedrock-Fluid-Veins.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Bedrock Fluid Veins
+---
+
+
+Bedrock fluid veins can be pumped using fluid drills.
+
+## Creating a Bedrock Fluid Vein
+
+
+```js title="fluid_veins.js"
+// In server events
+GTCEuServerEvents.fluidVeins(event => {
+
+    event.add('start:abydos_titanite_rich_magma_deposit', vein => {
```

Comment:
best to modify the example names and ores to be generic rather than pack specific